Makefile

```diff
@@ -75,13 +75,20 @@ integration-test-with-docker: prepare-docker-runner-image generate-env-file ## R
 		${DOCKER_BUILDER_IMAGE_NAME} \
 		make integration-test
 
+
+.PHONY: publish-to-rubygems
+publish-to-rubygems:
+	$(if ${GEM_HOST_API_KEY},,$(error Must specify GEM_HOST_API_KEY))
+	gem build notifications-ruby-client.gemspec --output=release.gem
```

Comment:
Seemed to all work fine. It took me a while to get it to work locally (bundler versions, then ruby versions) and then I ended up building and running this in a docker container to verify it works.

Makes me realise docker is helpful for local development. Probably no need to change this or add a publish-to-rubygems-with-docker method as it's pretty simple but just thought I'd point out what I'd found hard.